### DISCUSSION TITLE
Add a missing step to the scripting additions wiki

### DISCUSSION THREAD

**Discussion by ghost** [asmvik/yabai#2294] [Ideas]
I spent a lot of time debugging and troubleshooting this issue and the fix is easy once you get it, i had to reformat my mac (thats how much i wanted to make it work lol) and start from scratch  just to find out the source of the bug which was commenting out the last line from the sudoers file.

I would like to propose an addition to the wiki to help out future developers interested in using Yabai to its fullest potential 💪

_Steps i took to get scripting additions working on boot_

1. First you have to partially disable SIP
https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection

2. install the scripting additions
https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition

3. uncomment the last line from the sudoers file
`sudo visudo -f /private/etc/sudoers`
it should read
`#includedir /private/etc/sudoers.d`

4. then reboot your mac

**Comment by 24unix** [comment]
You should also file a bug report to the sudo coders, that that it the stupidest possible solution to include a file.
A hash in the first line is for the shebang, everything else is just a comment.

**Reply by ghost** [reply]
Haha yeah its an ancient relic from a long time ago...

I'll add it to my list of bug reports lol
